docs/reference/benchmarks.md

```diff
@@ -0,0 +1,40 @@
+| harfbuzz-1.3.2              | fuzz-target            | False           | 58                    | 10021                  | 6.24      |
+| irssi_server-fuzz           | server-fuzz            | True            | 895                   | 37455                  | 15.3      |
+| jsoncpp_jsoncpp_fuzzer      | jsoncpp_fuzzer         | True            | 0                     | 5536                   | 5.96      |
+| lcms-2017-03-21             | fuzz-target            | False           | 1                     | 6959                   | 6.11      |
+| libjpeg-turbo-07-2017       | fuzz-target            | False           | 1                     | 9586                   | 6.4       |
+| libpcap_fuzz_both           | fuzz_both              | False           | 0                     | 8149                   | 6.14      |
+| libpng-1.2.56               | fuzz-target            | False           | 1                     | 2991                   | 5.8       |
+| libxml2-v2.9.2              | fuzz-target            | False           | 0                     | 50461                  | 8.23      |
+| mbedtls_fuzz_dtlsclient     | fuzz_dtlsclient        | False           | 1                     | 10942                  | 6.67      |
+| openssl_x509                | x509                   | True            | 2241                  | 45989                  | 18.14     |
+| openthread-2019-12-23       | fuzz-target            | False           | 0                     | 17932                  | 6.72      |
+| php_php-fuzz-parser         | php-fuzz-parser        | True            | 2782                  | 123767                 | 15.57     |
+| proj4-2017-08-14            | fuzz-target            | False           | 44                    | 6156                   | 6.17      |
+| re2-2014-12-09              | fuzz-target            | False           | 0                     | 6547                   | 6.02      |
+| sqlite3_ossfuzz             | ossfuzz                | False           | 1258                  | 45136                  | 7.9       |
+| systemd_fuzz-link-parser    | fuzz-link-parser       | False           | 6                     | 53453                  | 5.91      |
+| vorbis-2017-12-11           | fuzz-target            | False           | 1                     | 5022                   | 6.0       |
+| woff2-2016-05-06            | fuzz-target            | False           | 62                    | 10923                  | 6.72      |
+| wpantund-2018-02-27         | fuzz-target            | False           | 8                     | 28888                  | 7.35      |
+| zlib_zlib_uncompress_fuzzer | zlib_uncompress_fuzzer | False           | 0                     | 875                    | 5.69      |
+
+Size is determined by the size of the libFuzzer build of the target.
+Number of program edges is the number of "guards" in the libFuzzer build of the target.
```

Comment:
consider: "number of instrumented program locations" ?

Reply:
just say "number of instrumented basic blocks". Edges are only known at fuzzing time given the instrumentation done. 

Seed and dictionary may or may not be used by the fuzzer, so maybe we should add a disclaimer about this. 

If we can provide where the seeds come from, or how they have been generated, that's also useful. Just bonus